wiki/Accuracy/th.md

```diff
@@ -23,49 +23,49 @@
 
 ### ![](/wiki/shared/mode/catch.png) osu!catch
 
 ![ความแม่นยำ = (จำนวนของ droplets + จำนวนของ drops + จำนวนของ fruits) / (จำนวนของ missed droplets + จำนวนของ missed drops + จำนวนของ missed fruits + จำนวนของ droplets + จำนวนของ drops + จำนวนของ fruits)](img/accuracy_catch.png "สูตรการคำนวนความแม่นยำของ osu!catch")
 
-ใน osu!catch, ความแม่นยำจะถูกคำนวนโดย ใช้จำนวนของ Hit objects ที่เก็บได้ทั้งหมดที่ไม่ใช่ Spinner , หารด้วยจำนวนของ hit objects ทั้งหมด. Hit objects ทุกอันมีค่าเท่ากัน; ยกเว้นกล้วย เพราะว่ามันเป็นส่วนหนึ่งของ Spinner
+ใน osu!catch ความแม่นยำจะถูกคำนวนโดย ใช้จำนวนของ Hit objects ที่เก็บได้ทั้งหมดที่ไม่ใช่ Spinner  หารด้วยจำนวนของ hit objects ทั้งหมด Hit objects ทุกอันมีค่าเท่ากัน; ยกเว้นกล้วย เพราะว่ามันเป็นส่วนหนึ่งของ Spinner
 
-*โน้ตสำหรับผู้ใช้ API: การที่จะคำนวนความแม่นยำใน osu!catch, จำนวนของ droplets อยู่ใน `count50` และจำนวนของ droplets ที่พลาดอยู่ใน `countkatu`.*
+*โน้ตสำหรับผู้ใช้ API: การที่จะคำนวนความแม่นยำใน osu!catch จำนวนของ droplets อยู่ใน `count50` และจำนวนของ droplets ที่พลาดอยู่ใน `countkatu`*
 
 ### ![](/wiki/shared/mode/mania.png) osu!mania
 
 ![ความแม่นยำ = (50 \* จำนวนของ 50s + 100 \* จำนวนของ 100s + 200 \* จำนวนของ 200s + 300 \* จำนวนของ 300s + 300 \* จำนวนของ MAXs) / 300(จำนวนของ 0s + จำนวนของ 50s + จำนวนของ 100s + จำนวนของ 200s + จำนวนของ 300s + จำนวนของ MAXs)](img/accuracy_mania.png "สูตรการคำนวนความแม่นยำของ osu!mania")
 
-ใน osu!mania, ความแม่นยำถูกคำนวนคล้ายกับ [osu!standard](#-osu!standard).
+ใน osu!mania ความแม่นยำถูกคำนวนคล้ายกับ [osu!standard](#-osu!standard)
 
 ## กราฟประสิทธิภาพ
 
 ![กราฟประสิทธิภาพ](img/performance_graph.jpg "กราฟประสิทธิภาพ")
 
 กราฟประสิทธิภาพ เป็นกราฟที่แสดงให้เห็นถึงประสิทธิภาพของผู้เล่น (โดยอ้างอิงจาก Life Bar) ระหว่างการเล่น (เวลา) ข้อมูลเพิ่มเติม สามารถดูได้โดยการนำ cursor ไปชี้บนกราฟ
 
-*โน้ต: ข้อมูลเพิ่มเติมสามารถดูได้เฉพาะ หลังจากเล่น Beatmap นั้น หรือ ดู replay หลังจากออก [หน้าแสดงผล](/wiki/results_screen), ข้อมูลส่วนนี้จะไม่ถูกเซฟ.*
+*โน้ต: ข้อมูลเพิ่มเติมสามารถดูได้เฉพาะ หลังจากเล่น Beatmap นั้น หรือ ดู replay หลังจากออก [หน้าแสดงผล](/wiki/results_screen) ข้อมูลส่วนนี้จะไม่ถูกเซฟ*
 
 ### ความแม่นยำ
 
-เมื่อนำ cursor ไปชี้บนกราฟประสิทธิภาพ, จะแสดง เรตติ้งของ `ความผิดพลาด` and `อัตราความไม่มั่นคง`.
+เมื่อนำ cursor ไปชี้บนกราฟประสิทธิภาพ จะแสดง เรตติ้งของ `ความผิดพลาด` and `อัตราความไม่มั่นคง`
 
-เพราะว่า ม้อด [DT](/wiki/DT) (Double Time) และ [HT](/wiki/HT) (Half Time) มีการนำมาใช้ ,ค่าของ ความผิดพลาด และ อัตราความไม่มั่นคง จะถูกคูณด้วยตัวคูณเดียวกันกับของแมป เพื่อที่จะได้ค่าแท้จริงของ DT, และหารผลลัพธ์ด้วย 1.5 ซึ่งคล้ายกับการคูณผลลัพธ์ด้วย 1.33 เมื่อเล่น HT
+เพราะว่า ม้อด [DT](/wiki/DT) (Double Time) และ [HT](/wiki/HT) (Half Time) มีการนำมาใช้ ,ค่าของ ความผิดพลาด และ อัตราความไม่มั่นคง จะถูกคูณด้วยตัวคูณเดียวกันกับของแมป เพื่อที่จะได้ค่าแท้จริงของ DT และหารผลลัพธ์ด้วย 1.5 ซึ่งคล้ายกับการคูณผลลัพธ์ด้วย 1.33 เมื่อเล่น HT
```

Comment:
```suggestion
เพราะว่า ม้อด [DT](/wiki/DT) (Double Time) และ [HT](/wiki/HT) (Half Time) มีการนำมาใช้ค่าของ ความผิดพลาด และ อัตราความไม่มั่นคง จะถูกคูณด้วยตัวคูณเดียวกันกับของแมป เพื่อที่จะได้ค่าแท้จริงของ DT และหารผลลัพธ์ด้วย 1.5 ซึ่งคล้ายกับการคูณผลลัพธ์ด้วย 1.33 เมื่อเล่น HT
```
I don't think this makes any sense (doesn't match the English original). Let's fix the stray comma for now.

Reply:
Will fix it in another PR